### PR TITLE
Added quotes

### PR DESCRIPTION
**Pull request by Shashwat-pd**
*updated quotes/nepali.json

Added 15 iconic poetry verses in the quotes of the Nepali language.

### CODE REVIEW

**Comment by Miodec** [comment]
Could you use longer versions of quotes shorter than 60 characters? Could you also translate the quotes?

File: frontend/static/quotes/nepali.json [hunk 1]
```diff
@@ -510,6 +510,107 @@
       "source": "लेस ब्राउन",
       "length": 80,
       "id": 84
+    },
+    {
+      "text": "के हो ठूलो जगतमा ।\nपसीना विवेक  उद्देश्य के लिनु ।\n उडी छुनु चन्द्र एक",
```
Comment (Shashwat-pd) [inline frontend/static/quotes/nepali.json:515]: What is in the big world? What is the purpose of sweating conscience?

File: frontend/static/quotes/nepali.json [hunk 2]
```diff
@@ -510,6 +510,107 @@
...
+      "source": "लक्ष्मी प्र देवकोटा",
+      "length": 67,
+      "id": 85
+    },
+    {
+      "text": "निद्राकी सखी शान्ति हुन् निदाएको बालक देख्दा शान्तिको सच्चा मुहार देखिन्छ",
```
Comment (Shashwat-pd) [inline frontend/static/quotes/nepali.json:521]: Sleeping love is peace. When you see a sleeping child, you see the true face of peace

File: frontend/static/quotes/nepali.json [hunk 3]
```diff
@@ -510,6 +510,107 @@
...
+      "source": "लक्ष्मी प्र देवकोटा",
+      "length": 74,
+      "id": 86
+    },
+    {
+      "text": "छन्द भनेको सच्चा अर्थमा वोल्नुको ढाँचा होस यसको माने छहरा हो, लहरा होइन",
```
Comment (Shashwat-pd) [inline frontend/static/quotes/nepali.json:527]: Poetry is a form of speaking in the true sense, it feels, not reads

File: frontend/static/quotes/nepali.json [hunk 4]
```diff
@@ -510,6 +510,107 @@
...
+      "source": "लक्ष्मी प्र देवकोटा",
+      "length": 71,
+      "id": 87
+    },
+    {
+      "text": "जनता जो सच्चा राष्ट्र हो, जो राष्ट्रको जीवनको नाडी र भावको गुन्जनं हो ।",
```
Comment (Shashwat-pd) [inline frontend/static/quotes/nepali.json:533]: The people who are the true citizen, are the pulse and emotion of the life of the nation.

File: frontend/static/quotes/nepali.json [hunk 5]
```diff
@@ -510,6 +510,107 @@
...
+      "source": "लक्ष्मी प्र देवकोटा",
+      "length": 71,
+      "id": 88
+    },
+    {
+      "text": "जीवनहरू. विन्दुमा सुन्दर देखिन्न, कला देखिन्छ, जीवनको सत्य यथार्थिक या व्यावहारिक हुन्छ,\nकलाका काल्पनिक हुन्छ ।",
```
Comment (Shashwat-pd) [inline frontend/static/quotes/nepali.json:539]: Lives. The point is not to look beautiful, art is to be seen, the truth of life is real or practical, art is fiction.

File: frontend/static/quotes/nepali.json [hunk 6]
```diff
@@ -510,6 +510,107 @@
...
+      "source": "लक्ष्मी प्र देवकोटा",
+      "length": 111,
+      "id": 88
+    },
+    {
+      "text": "हिंड्दा–हिंड्दै केही सम्झेर बटुवा बाटोमा हाँसेझैं\nतिमी हाँस्ता यस्तो लाग्छ प्रिय !\nतिमी मेरो साटोमा हाँसेझैं ।",
```
Comment (Shashwat-pd) [inline frontend/static/quotes/nepali.json:545]: I reminisce your laughter and I laugh whilst in the middle of the road. Your laugh my love, feels like you're laughing for me

File: frontend/static/quotes/nepali.json [hunk 7]
```diff
@@ -510,6 +510,107 @@
...
+      "source": "भूपी शेरचन हिंड्दा हिंड्दै",
+      "length": 107,
+      "id": 89
+    },
+    {
+      "text": "पाउँ बस्न मरेर आखिर गई स्वर्गीय त्यै ठाउँ मानेपालीहरु छन् मरीकन सबै जन्मिरहेका जहाँ",
```
Comment (Shashwat-pd) [inline frontend/static/quotes/nepali.json:551]: Nepali die and go to heaven and realized heaven was where they were actually born.

File: frontend/static/quotes/nepali.json [hunk 8]
```diff
@@ -510,6 +510,107 @@
...
+      "source": "बालकृश्न सम इच्छा",
+      "length": 83,
+      "id": 90
+    },
+    {
+      "text": "सत्यका लागि त्याग गर्न नसक्नेहरू पनि वीर होइनन",
```
Comment (Shashwat-pd) [inline frontend/static/quotes/nepali.json:557]: Those who cannot sacrifice for the truth are not heroes

File: frontend/static/quotes/nepali.json [hunk 9]
```diff
@@ -510,6 +510,107 @@
...
+      "length": 46,
+      "id": 91
+    },{
+      "text": "जून फूल मैले चाहेको थिएँ कसैले टिपेछ\nसौन्द्रय बनि त्यो सधै मेरो आँखामा लुकेछ",
+      "source": "नारायण गोपाल",
+      "length": 83,
+      "id": 92
+    },
+    {
+      "text": "परालको निम्ती कल्चौँडाको दुध बेच्छन् गाईहरू\nचाराको निम्ती गीत बेच्छन् मिठो बोल्ने सुगाँहरू",
```
Comment (Shashwat-pd) [inline frontend/static/quotes/nepali.json:568]: Cows sell their child's milk for food. Birds sell their beautiful vocals.

File: frontend/static/quotes/nepali.json [hunk 10]
```diff
@@ -510,6 +510,107 @@
...
+      "source": "भैरव अर्याल",
+      "length": 89,
+      "id": 93
+    },
+    {
+      "text": "प्राप्तिभन्दा ध्येय ठूलो हो, पुग्नुभन्दा आशा र खोज महत्वपूर्ण हो",
```
Comment (Shashwat-pd) [inline frontend/static/quotes/nepali.json:574]: The goal is greater than achievement, hope and discovery are more important than achievement

File: frontend/static/quotes/nepali.json [hunk 11]
```diff
@@ -510,6 +510,107 @@
...
+      "source": "लक्ष्मी प्र देवकोटा",
+      "length": 64,
+      "id": 94
+    },
+    {
+      "text": "हुँदैन बिहान मिर्मिरेमा तर झरेर नगए\nबन्दैन मुलुका दुई-चार सपूत मरेर नगए",
```
Comment (Shashwat-pd) [inline frontend/static/quotes/nepali.json:580]: The mornings come only after the stars fall off. Nations are revolutionized only after the death of a couple of heroes.

File: frontend/static/quotes/nepali.json [hunk 12]
```diff
@@ -510,6 +510,107 @@
...
+      "source": "भैरव अर्याल",
+      "length": 70,
+      "id": 95
+    },
+    {
+      "text": "अगुल्टोले हनेको कुकुर बिजुली चम्कदा तर्सिन्छ",
```
Comment (Shashwat-pd) [inline frontend/static/quotes/nepali.json:586]: A dog struck by lightning is frightened by electricity

File: frontend/static/quotes/nepali.json [hunk 13]
```diff
@@ -510,6 +510,107 @@
...
+      "source": "उखान टुक्का",
+      "length": 89,
+      "id": 96
+    },
+    {
+      "text": "सुघरी स्वास्निको पोइ मोटाउछ,\nफोहोरी स्वास्निको कुकुर मोटाउछ",
```
Comment (Shashwat-pd) [inline frontend/static/quotes/nepali.json:592]: Clean wife's husbands gets fat, dirty wife's dogs get fat

File: frontend/static/quotes/nepali.json [hunk 14]
```diff
@@ -510,6 +510,107 @@
...
+      "source": "उखान टुक्का",
+      "length": 62,
+      "id": 97
+    },
+    {
+      "text": "घनघोर दुःख सागर संसार जान भाई,\nनगर घमण्ड कहिले मर्नु छ हामीलाई",
```
Comment (Shashwat-pd) [inline frontend/static/quotes/nepali.json:598]: A sea of deep sorrow to go to the world, brother, city pride, when do we have to die

File: frontend/static/quotes/nepali.json [hunk 15]
```diff
@@ -510,6 +510,107 @@
...
+      "source": "लक्ष्मी प्र देवकोटा",
+      "length": 62,
+      "id": 98
+    },
+    {
+      "text": "ओरालो लागेको गाई लाई राम राम भन्न सकिन्छा काध थप्न सकिदैन । ",
```
Comment (Shashwat-pd) [inline frontend/static/quotes/nepali.json:604]: A falling cow should be said sorry not provided hand.

File: frontend/static/quotes/nepali.json [hunk 16]
```diff
@@ -510,6 +510,107 @@
...
+      "source": "उखान टुक्का",
+      "length": 60,
+      "id": 99
+    },
+    {
+      "text": "जरुर साथी म पागल !\nयस्तै छ मेरो हाल ।\nम शब्दलाई देख्दछु !\nदृश्यलाई सुन्दछु !\nबासनालाई संबाद लिन्छु ।",
```
Comment (Shashwat-pd) [inline frontend/static/quotes/nepali.json:610]: Definitely my friend, I'm crazy! This is how I am now. I see the word! I hear the scene! I will talk to smell.

File: frontend/static/quotes/nepali.json (hunk 13 above)
Comment (Bruception) [reply frontend/static/quotes/nepali.json:592]: I’m not sure if this quote is acceptable.

Comment (Miodec) [reply frontend/static/quotes/nepali.json:592]: Remove please

File: frontend/static/quotes/nepali.json [hunk 17]
```diff
@@ -510,6 +510,107 @@
...
+      "id": 95
+    },
+    {
+      "text": "अगुल्टोले हनेको कुकुर बिजुली चम्कदा तर्सिन्छ",
+      "source": "उखान टुक्का",
+      "length": 89,
+      "id": 96
+    },
+    {
+      "text": "सुनको थैला हातको मैला के गर्नु धनले,\nसाग र सिस्नु खाएको बेस आनन्द मनले",
```
Comment (Shashwat-pd) [inline frontend/static/quotes/nepali.json:592]: What to do with the bag of gold that feels like the filth of the hand, pleasure lies in eating vegetables with family

File: frontend/static/quotes/nepali.json [hunk 18]
```diff
@@ -510,6 +510,107 @@
...
+      "id": 89
+    },
+    {
+      "text": "पाउँ बस्न मरेर आखिर गई स्वर्गीय त्यै ठाउँ मानेपालीहरु छन् मरीकन सबै जन्मिरहेका जहाँ",
+      "source": "बालकृश्न सम इच्छा",
+      "length": 83,
+      "id": 90
+    },
+    {
+      "text": "क्षेत्रीको छोरो यो पाउ छुन्छ, घिनले छुँदैन,\nमानिस ठूलो दिलले हुन्छ जातले हुँदैन",
```
Comment (Shashwat-pd) [inline frontend/static/quotes/nepali.json:557]: Chhetri's son touches this foot, not with disgust, man is with a big heart, not with caste

**Comment by Shashwat-pd** [comment]
@Miodec I believe it's good enough now.

**Comment by Miodec** [comment]
Quote IDs are not unique and length fields are incorrect

**Comment by Shashwat-pd** [comment]
@Miodec Good enough to merge now, I believe

**Comment by Shashwat-pd** [comment]
@Miodec  Good enough to merge now I believe

**Pull request merged**